generator.ts

```diff
@@ -0,0 +1,254 @@
+import { promises as fsPromises } from 'fs';
+
+import _ from 'lodash';
+import yaml from 'js-yaml';
+
+import { init } from './init';
+import {
+  getResourceSchema,
+  getResultSchema,
+  getSetResultSchema,
+  getPostBodySchema,
+  getPatchBodySchema,
+} from './schemas';
+
+/**
+ * Gets the prefix of the resource schema in components/schemas
+ *
+ * @param resourceName
+ * @returns The prefix
+ */
+const getResourceSchemaPrefix = (resourceName: string) => _.capitalize(resourceName);
+
+/**
+ * Gets the method for a given endpoint. Example: `getById` -> `get`
+ *
+ * @param endpoint
```

Comment:
`getById` doesn't' sound like an "endpoint" to me, it sounds more like an operation. What do you think?

Reply:
Yeah, I like this better than endpoint.